File: public_html/CPP/tabliczka_mnozenia.cpp
/*
 * tabliczka_mnozenia.cpp
 * 
 */


#include <iostream>
#include <iomanip>

using namespace std;

void tabliczka(int a, int b)
{
    
    //int kolumna, wiersz;
    
    for (int i = 1; i <= a; i++)
    {
        for (int j = 1; j <= b; j++)
        {   
            cout << setw(4) << i * j;
        }
        
        cout << endl;
    }
}

int main(int argc, char **argv)
{   
    int a, b;  //deklaracja
    a = b = 0;  //inicjacja
    
    cout << "Podaj zakres liczb w kolumnie: ";
    cin >> a;
        
    cout << "Podaj zakres liczb w wierszu: ";
    cin >> b;
    
    tabliczka(a, b);
    	
	return 0;
}

File: python/tabliczka_mnozenia.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  tabliczka_mnozenia.py
#

def tabliczka(a, b):

    for kolumna in range(1, a + 1): #pętla zewnętrzna
        for wiersz in range(1, b + 1): #pętla wewnętrzna
            print("{:>3} ".format(wiersz*kolumna), end='')
        print()

def main(args):

    a = int(input("Podaj zakres liczb w kolumnie: "))
    b = int(input("Podaj zakres liczb w wierszu: "))
    
    tabliczka(a, b)
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
